test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -349,47 +317,47 @@ private static IRazorEngineFeature GetMetadataReferenceFeature(IEnumerable<Metad
             };
         }
 
-        private static IEnumerable<MetadataReference> CreateAppCodeReferences(string appCode, IEnumerable<MetadataReference> shimReferences)
+        private class NormalizedDefaultImportFeature : RazorProjectEngineFeatureBase, IImportProjectFeature
```

Comment:
Move to testcommon?

Reply:
No. This is specific to Mvc.Extensions. Razor.Language integration tests don't have this problem.

Reply:
Because they don't use imports?

Reply:
This is to normalize the default imports. The imports we use in addition to this follows it's own convention `Name_Imports0.cshtml`. I have a custom import feature for that as well which exists in Test.Common.